### ISSUE TITLE
scala-cli: AirSpec test hungs after completion

### ISSUE THREAD

**Issue by xerial** [wvlet/airframe#3049]
When using AirSpec with scala-cli, the process hungs:

Build.test.scala
```scala
//> using test.dep org.wvlet.airframe::airspec:23.7.0
//> using testFramework "wvlet.airspec.Framework"
import wvlet.airspec.AirSpec

class BuildTest extends AirSpec:
  test("foo") {
    // TODO: hangs after completion
  }
```

```
$ scala-cli test .
Compiling project (test, Scala 3.3.0, JVM)
Compiled project (test, Scala 3.3.0, JVM)
BuildTest:
 - foo 6.82ms <----------- hungs here 
```

Probably because async test support of AirSpec is related.

**Comment by xerial** [comment]
It looks like thread manager is not closed after test completion: 
https://github.com/wvlet/airframe/blob/c9352ab76ceec7da1898e5eb453ac706a556bf99/airspec/.jvm/src/main/scala/wvlet/airspec/Compat.scala#L35

**Comment by xerial** [comment]
Fixed in airspec 23.7.1
